web/src/app/shared/models/project-token.model.ts

```diff
@@ -0,0 +1,14 @@
+export interface IProjectTokenModel {
+  uid: string;
+  name: string;
+}
+
+export class ProjectTokenModel {
+  uid: string;
+  name: string;
+
+  constructor(obj?: IProjectTokenModel) {
```

Comment:
This is an interesting concept, as we don't use it at the moment, I think the interface is not required, and having the 2 variables as parameters is more consistent. 

This is definitely something to consider going forwards as it is cleaner 👍 - if we do go down this route, we would have to raise a ticket to do it application wide at the same time

Reply:
I forgot to implement the interface in class

Reply:
I like the idea, but as this isn't something we do yet, it feels wrong to have it different and inconsistent. 

As mentioned, I think we should be consistent here even if this style is better, and if we do this style it needs to be implemented in another PR application wide. I raised issue #1337 where this style can be listed

> This is an interesting concept, as we don't use it at the moment, I think the interface is not required, and having the 2 variables as parameters is more consistent.
> This is definitely something to consider going forwards as it is cleaner 👍 - if we do go down this route, we would have to raise a ticket to do it application wide at the same time